web/gds-user-ui/package.json

```diff
@@ -132,6 +134,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.18.0",
+    "@badeball/cypress-cucumber-preprocessor": "^17.2.0",
```

Comment:
This looks a little different from Ensign's setup because the repository moved. Here's a link mentioning that: https://github.com/badeball/cypress-cucumber-preprocessor/issues/689

I want to note that this dependency only works with Node versions 18+. I upgraded to version 18.16.0 since that's the recommended version now: https://nodejs.org/en